Library/Homebrew/config.rb

```diff
@@ -12,9 +12,6 @@ class MissingEnvironmentVariables < RuntimeError; end
 #
 # @api private
 module EnvVar
-  extend T::Sig
-
-  sig { params(env: String).returns(String) }
```

Comment:
What happens if you add this back? Does it error-out in a sufficiently easy to understand version? I wonder if it's worth having a comment here noting the `.rbi` file?

Reply:
It errors because `require "sorbet-runtime"` or `require "sorbet-runtime-stub"` hasn't been run yet so `T` doesn't exist.

Reply:
Yeh, I figured that's _why_ it errored but I was wondering _how_ it errored i.e. if the messaging is obvious enough that people will understand what they've done wrong here.

Reply:
```
Traceback (most recent call last):
	5: from /usr/local/Homebrew/Library/Homebrew/brew.rb:31:in `<main>'
	4: from /usr/local/Homebrew/Library/Homebrew/brew.rb:31:in `require_relative'
	3: from /usr/local/Homebrew/Library/Homebrew/global.rb:75:in `<top (required)>'
	2: from /usr/local/Homebrew/Library/Homebrew/global.rb:75:in `require'
	1: from /usr/local/Homebrew/Library/Homebrew/config.rb:14:in `<top (required)>'
/usr/local/Homebrew/Library/Homebrew/config.rb:15:in `<module:EnvVar>': uninitialized constant EnvVar::T (NameError)
```